src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,17 @@
+package duke.command;
+
+import duke.DukeExceptions;
+import duke.TodoList;
+
+public class MarkCommand extends Command{
+    private TodoList todoList;
+    private int index;
+
+    public MarkCommand(TodoList todoList, int index) {
+        this.todoList = todoList;
+        this.index = index;
+    }
+
+    @Override
+    public void execute() throws DukeExceptions { todoList.mark(index); }
```

Comment:
Just a small suggestion✌🏼 Since the MarkCommand and UnmarkCommand classes do the same thing except the execute functions maybe you want to consider just using 1 class that handles all marking related commands with a check to determine whether to mark or unmark the task